CHANGELOG.md

```diff
@@ -8,15 +8,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 - [2635](https://github.com/FuelLabs/fuel-core/pull/2635): Add metrics to gas price service
+- [2553](https://github.com/FuelLabs/fuel-core/pull/2553): Scaffold global merkle root storage crate.
+- [2598](https://github.com/FuelLabs/fuel-core/pull/2598): Add initial test suite for global merkle root storage updates.
```

Comment:
Would be nice not to touch old changes in changelog=)

Reply:
yeah it got auto formatted